release/models/containerz/openconfig-containerz.yang

```diff
@@ -0,0 +1,139 @@
+  }
+
+
+
+  // gRPC server related definitions.
+  // Success/failure counters.
+  grouping grpc-server-containerz-counters  {
+    description
+      "A collection of counters that were collected while evaluating
+      access to the gRPC server.";
+
+    container counters {
+      config false;
+      description
+      "A collection of counters that were collected by the gRPC during
+      the authentication process.";
+
+      leaf connection-rejects {
+        type oc-yang:counter64;
+        description
+        "The total number of times that gRPC clients have failed
+        in establishing a connection to the server.";
+      }
+      leaf last-connection-reject {
+        type oc-types:timeticks64;
+        description
+        "A timestamp of the last time a gRPC client failed
+        in establishing a connection to the server.";
+      }
+      leaf connection-accepts {
+        type oc-yang:counter64;
+        description
+        "The total number of times that gRPC clients have succeeded
+        in establishing a connection to the server.";
+      }
+      leaf last-connection-accept {
+        type oc-types:timeticks64;
+        description
+        "A timestamp of the last time a gRPC client succeeded
+        in establishing a connection to the server.";
+      }
+      leaf running-containers {
+        type oc-yang:counter64;
+        description
+        "The number of containers currently running on the system.";
+      }
+    }
+  }
+
+  grouping grpc-server-containerz-state {
+    description
+      "gRPC server containerz freshness-related data.";
+
+    container containers {
+      list container {
+        key "name";
+
+        leaf name {
+          type string;
+        }
+
+        leaf created-on {
+          type oc-types:timeticks64;
+          description
+           "The timestamp when the container was created.
+            The value is the timestamp in nanoseconds relative to
+            the Unix Epoch (Jan 1, 1970 00:00:00 UTC).";
+        }
+
+        leaf restarts {
+           type oc-yang:counter64;
+           description
+             "The number of times the container was restarted.";
+        }
+
+        leaf version {
+          type string;
+          description
+            "The version of the of the container. This corresponds to the
+             tag of of the image associated to this container.";
+        }
+      }
+    }
+  }
+
+  // Augments section.
+
+  augment "/oc-sys:system/oc-sys-grpc:grpc-servers/oc-sys-grpc:grpc-server/" +
+          "oc-sys-grpc:state" {
+    when "../config[contains(services, 'oc-containerz:CONTAINERZ')]/enable = 'true'";
+    description
+      "A gRPC server containerz freshness information.";
+
+    uses grpc-server-containerz-state;
+  }
+
+  augment "/oc-sys:system/oc-sys-grpc:grpc-servers/oc-sys-grpc:grpc-server/" +
+          "oc-sys-grpc:state" {
+    when "../config[contains(services, 'oc-containerz:CONTAINERZ')]/enable = 'true'";
```

Comment:
This identity does not exist.  For new gRPC services, there is current way and potential future for expansion but you'll want to at a min follow what you should see for P4RT and GRIBI